js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -62,15 +63,17 @@ export default function AssetGroup( { campaign } ) {
 	const isCreation = ! campaign;
 	const { isValidForm, handleSubmit, adapter, values } =
 		useAdaptiveFormContext();
+	const { data: countryCodes } = useTargetAudienceFinalCountryCodes();
 	const { isValidAssetGroup, isSubmitting, isSubmitted, submitter } = adapter;
 	const currentAction = submitter?.dataset.action;
+	const audiences = isCreation ? countryCodes : campaign.displayCountries;
```

Comment:
This variable is only used within the scope of `recordSubmissionClickEvent` function. How about moving the variable declaration into the function?

Reply:
👍